src/components/DuffelNGSView/lib/sort-ngs-rows.ts

```diff
@@ -19,6 +19,7 @@ export const sortNGSRows = (
     if (aAmount && bAmount) {
       return sortDirection === "asc" ? aAmount - bAmount : bAmount - aAmount;
     }
+    if (aAmount) return -1;
```

Comment:
should if also have something like this:
```suggestion
    if (aAmount) return -1;
    if (bAmount) return 1;
```
?